Dockerfile.rhel

```diff
@@ -0,0 +1,22 @@
+# This Dockerfile is used by CI to publish odo binary artifacts.
+# It builds an image containing the Mac, Win and Linux version of odo binary on the
+# OpenShift golang image.
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
+LABEL Description="This image is used to build odo(Developer-focused CLI for OpenShift) \
+executables for Windows, macOS and linux" Version="1.0"
+
+COPY . /go/src/github.com/openshift/odo
+WORKDIR /go/src/github.com/openshift/odo
+RUN go get github.com/mitchellh/gox &&\
```

Comment:
this is not going to work in a closed environment.

We should update our `make cross` target to not require gox,

Reply:
> this is not going to work in a closed environment.

@kadel Can you please point out the reasons ?

Reply:
@kadel is right we are again pulling something from the internet, maybe we can explore the option of replicating with `GOOS` env?

Reply:
or we host gox in our repo

Reply:
Done

Reply:
Shouldn't be that line removed now?

Reply:
```gox``` dependency removed